src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/OperationExtensions.cs

```diff
@@ -152,6 +152,29 @@ public static ValueUsageInfo GetValueUsageInfo(this IOperation operation)
                         return ValueUsageInfo.Read;
                 }
             }
+            else if (operation.Parent is IReturnOperation)
+            {
+                var containingMethod = TryGetContainingAnonymousFunctionOrLocalFunction(operation)
```

Comment:
@mavasani can we not find the containing method from the op-tree?  should we be able to get that?

Reply:
Nope, that information is not present in the Operation tree.